libwebui.so.1

```diff
@@ -0,0 +1 @@
+libwebui.so.1.0.0
```

Comment:
Thanks for the updates, looks like you forgot to remove the libwebui.so files, and some of the unused lines in the css.